test.md

```diff
@@ -0,0 +1,7 @@
+# Object file
+*.o
+
+# Ada Library Information
+*.ali
+# This file should be ignored by git
+*.md
```

Comment:
The test.md file should be ignored by git

Reply:
Just as I added in the previous comment, WHY test.md file should be ignored is important here as well. Also, try to present it in a polite way so it does not sound like you're issuing a command

In addition, from a beginner's standpoint, I would find the statement a bit confusing.
"I think you should add `test.md` file to `.gitignore` " followed by the reason would appeal more.